doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1729,7 +1729,7 @@ performance issue.
         M /\ L /\ K /\ J /\ I /\ H /\ G /\ F /\ E /\ D /\ C /\ B /\ A).
    Proof.
 
-.. coqtop:: all
+.. coqtop:: all warn
```

Comment:
What about replacing `omega` with `lia` in this example. Will this break the example because `lia` performs much better. cc @JasonGross who is probably the author of this example.

Reply:
IMHO, we can leave this example as-is for now. This does not prevent Jason or anyone to rework it in a separate PR.

Reply:
OK to leave `omega` in there, but you should locally disable the warning. Currently, the output contains:

```
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    Toplevel input, characters 0-4:
    > tac.
    > ^^^^
    Warning: omega is deprecated since 8.12; use “lia” instead.
    [omega-is-deprecated,deprecated]
    No more subgoals.
```

It seriously undermine the readability of the example.

Reply:
Done.